### ISSUE TITLE
Type of the output of `dict_list`

### ISSUE THREAD

**Issue by mancellin** [JuliaDynamics/DrWatson.jl#42]
Thank you for your work on this tool!

I'd like to report a small inconsistency with the output of `dict_list`. Whenever possible, I'd like the result to be a list of `Dict{Symbol,Any}` or `Dict{String,Any}` instead of a list of `Dict{Any,Any}`.
Here is an example in which it is not the case:
```julia
julia> test_matrix = Dict(:a => [1, 2], :b => ["x", "y"])
Dict{Symbol,Array{T,1} where T} with 2 entries:
  :a => [1, 2]
  :b => ["x", "y"]

julia> dict_list(test_matrix)
4-element Array{Dict{Any,Any},1}:
 Dict(:a=>1,:b=>"x")
 Dict(:a=>2,:b=>"x")
 Dict(:a=>1,:b=>"y")
 Dict(:a=>2,:b=>"y")
```

It works fine in many other cases such as [the example in the doc](https://juliadynamics.github.io/DrWatson.jl/dev/real_world/#Preparing-runs-1) or this one:
```julia
julia> test_matrix = Dict(:a => [1, 2], :b => [8, 9])
Dict{Symbol,Array{Int64,1}} with 2 entries:
  :a => [1, 2]
  :b => [8, 9]

julia> dict_list(test_matrix)
4-element Array{Dict{Symbol,Any},1}:
 Dict(:a=>1,:b=>8)
 Dict(:a=>2,:b=>8)
 Dict(:a=>1,:b=>9)
 Dict(:a=>2,:b=>9)
```
I tried some variants but I couldn't figure out exactly in which cases it returns `Dict{Any,Any}` or `Dict{Symbol,Any}`.

**Comment by Datseris** [comment]
The recent PR #41 improved this; can you check again and tell me whether the issue still persists in version `0.2.1` ?

**Comment by mancellin** [comment]
Ah, I missed that PR. Indeed, the problem has been solved in version `0.2.1` !
